components/local-app/pkg/auth/auth.go

```diff
@@ -75,12 +106,12 @@ func ValidateToken(client gitpod.APIInterface, tkn string) error {
 
 // SetToken returns the persisted Gitpod token
 func SetToken(host, token string) error {
-	return keyring.Set(keyringService, host, token)
```

Comment:
would it still be `gitpod-io` for local companion?

Reply:
https://github.com/gitpod-io/gitpod/pull/18971/commits/af605d87b4228ab269986f52e5581c9395a0e791